src/components/editor/property/InputLookupQA.jsx

```diff
@@ -227,16 +232,26 @@ InputLookupQA.propTypes = {
     valueConstraint: PropTypes.shape({
       useValuesFrom: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
     }),
-  }).isRequired,
+  }),
   reduxPath: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
   displayValidations: PropTypes.bool,
 }
 
 const mapStateToProps = (state, props) => {
-  const result = getProperty(state, props)
+  const reduxPath = props.reduxPath
+  const resourceTemplateId = reduxPath[reduxPath.length - 2]
+  const propertyURI = reduxPath[reduxPath.length - 1]
   const displayValidations = getDisplayValidations(state)
```

Comment:
I think for now calling `getDisplayValidations`, `getPropertyTemplate`, and `getLookupConfigItems` directly from reducers/index.js module is fine. A future rewrite would have a call functions in the `actions/index.js` instead.